File: LeetCode/266-palindrome-permutation/ex_266.cpp
class Solution {
public:
    bool canPermutePalindrome(string s) {
        set<char> charSet;
        for (auto c : s) {
            if (charSet.find(c) != charSet.end())
                charSet.erase(c);
            else
                charSet.insert(c);
        }
        return charSet.size() < 2;
    }
};

File: LeetCode/266-palindrome-permutation/ex_266.py
class Solution(object):
    def canPermutePalindrome(self, s):
        """
        :type s: str
        :rtype: bool
        """
        myset = set()
        for c in s:
            if c in myset:
                myset.remove(c)
            else:
                myset.add(c)
        return len(myset) < 2